<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReassemblyViewMain</class>
 <widget class="QMainWindow" name="ReassemblyViewMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <widget class="QGroupBox" name="workbenchBox">
       <property name="title">
        <string>ReAssembly Workbench</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="originalFragmentsLabel">
          <property name="text">
           <string>Original Fragments</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="originalFragmentsView">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="candidatePaLabel">
          <property name="text">
           <string>Candidate Partial Assemblies</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="canditatePaView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveAsReAssemblyGroup">
          <property name="text">
           <string>Save as ReAssembly Group</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="Reassembly3DWidget" name="reassembly3DWidget" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionLoad_ReAssembly_File">
   <property name="text">
    <string>Load ReAssembly File</string>
   </property>
  </action>
  <action name="actionReAssembly_Demo">
   <property name="text">
    <string>ReAssembly Demo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Reassembly3DWidget</class>
   <extends>QWidget</extends>
   <header>reassembly/reassembly3dwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
